src/ImmutableObjectGraph.CodeGeneration.Roslyn/CodeGen.cs

```diff
@@ -166,6 +166,18 @@ private void MergeFeature(FeatureGenerator featureGenerator)
 
             partialClass = this.mergedFeatures.Aggregate(partialClass, (acc, feature) => feature.ProcessApplyToClassDeclaration(acc));
             var outerMembers = SyntaxFactory.List<MemberDeclarationSyntax>();
+            if ( applyTo.Parent.Kind() == SyntaxKind.ClassDeclaration )
+            {
+                var parent = applyTo.Parent as ClassDeclarationSyntax;
+
+                while ( parent != null )
```

Comment:
Your whitespace rules don't match VS defaults. Can you adjust this to match the surrounding code? For instance, no space just inside your parentheses.